packages/jsdoc-plugins/lib/statics-augmentor/augmentor.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * @license Copyright (c) 2017, CKSource - Frederico Knabben. All rights reserved.
+ * Licensed under the terms of the MIT License (see LICENSE.md).
+ */
+
+'use strict';
+
+/**
+ * @see http://usejsdoc.org/about-plugins.html
+ */
+
+const StaticsAugmentor = require( './statics-augmentor' );
```

Comment:
A file should be called like the class it exports (but lowercase) – `staticsaugmentor.js`.